src/views/darkroom.c

```diff
@@ -75,6 +75,14 @@
 #define G_SOURCE_FUNC(f) ((GSourceFunc) (void (*)(void)) (f))
 #endif
 
+typedef enum image_size_t {
+  SIZE_SMALL,
+  SIZE_MEDIUM,
+  SIZE_LARGE
+} image_size_t;
+
+#define SCALE(x) x / ppd
```

Comment:
`((x) / ppd)`

Otherwise calling with SCALE(v*2) won't give you the proper result.

Reply:
done